msg/RadarReturn.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
+float32 azimuth                          # Angle (in radians) in the azimuth plane between the sensor and the detected point.
+                                         #   Negative angles are anticlockwise from the sensor.
```

Comment:
Shouldn't this be positive angles are anticlockwise from the sensor?
According to [REP 103](https://www.ros.org/reps/rep-0103.html), `By the right-hand rule, the yaw component of orientation increases as the child frame rotates counter-clockwise`.

Reply:
I think this is poorly defined since clockwise wrt a front view or a rear view...? I agree this should be in line with REP103

Reply:
IIUC, REP-103 defines coordinate systems to be body-local. So clockwise from the perspective of the body-local coordinate system.

Reply:
OK, so should we just remove this comment? Perhaps replace it with one at the top of the message saying that the frames should conform with REP-103?

Reply:
Really great point @shrijitsingh99

Reply:
> OK, so should we just remove this comment? Perhaps replace it with one at the top of the message saying that the frames should conform with REP-103?

you could keep ~~the~~ a comment, update it so it aligns with REP-103 *and* refer to REP-103.

Just to avoid questions like the one here in this comment-thread in the future.

Reply:
Fair enough. @radarAaron any disagreements?

Reply:
agreed.